crates/bevy_window/src/window.rs

```diff
@@ -625,7 +625,8 @@ pub struct WindowDescriptor {
     /// macOS X transparent works with winit out of the box, so this issue might be related to: <https://github.com/gfx-rs/wgpu/issues/687>
     /// Windows 11 is related to <https://github.com/rust-windowing/winit/issues/2082>
     pub transparent: bool,
-    #[cfg(target_arch = "wasm32")]
+    /// This field is only available in wasm. Can take a CSS selector to an HTML canvas
+    #[cfg(any(target_arch = "wasm32", doc))]
```

Comment:
Maybe use `#[doc(cfg(target_arch = "wasm32"))]`? I'm not sure if it is stable already, but if not you could gate it to docs.rs.

Reply:
It's unstable: https://doc.rust-lang.org/rustdoc/unstable-features.html#doccfg-recording-what-platforms-or-features-are-required-for-code-to-be-present
Do you know if it or https://doc.rust-lang.org/rustdoc/unstable-features.html#doc_auto_cfg-automatically-generate-doccfg would be stabilised soon?

I'm not sure how to gate it for docs.rs, it seems it's only available through a var env in a build.rs script: https://docs.rs/about/builds#detecting-docsrs